app/components/earthworks/also_available_component.html.erb

```diff
@@ -0,0 +1,8 @@
+<h2 class="h4">Also available at</div>
```

Comment:
@peetucket this is what is breaking the layout. It should read:
`<h2>Also available at</h2>`
It shouldn't close the wrapping `</div>`

Reply:
thanks - fixed closing tag (but left `class="h4"`, which I think it still needed?)